### PR TITLE
[TwigComponent] Allow trailing coma in "props" tags

### LINKED ISSUE #1286: [TwigComponent] {% props syntax should allow dangling commas
Hi!

This currently errors:

```
{% props
    allowSmallWidth=false,
    closeButton=false,
%}
```

The problem is the dangling comma at the end of the list. If the last `,` is removed, this works fine. We should be a bit more friendly and allow this :).

Cheers!

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | not really
| Issues        | Fix #1286
| License       | MIT

Allow trailing comas in props tag

```twig
{% props 
    foo = 'foo',
    bar = 'bar',
%}
```

### CODE REVIEW

File: src/TwigComponent/tests/Integration/ComponentExtensionTest.php [hunk 1]
```diff
@@ -208,6 +208,14 @@ public function testComponentPropsOverwriteContextValue(): void
         $this->assertStringContainsString('<p>foo</p>', $output);
     }
 
+    public function testComponentPropsWithTrailingComa(): void
+    {
+        $output = self::getContainer()->get(Environment::class)->render('anonymous_component_props_trailing_coma.html.twig');
```
Comment (jdreesen) [inline src/TwigComponent/tests/Integration/ComponentExtensionTest.php]:
```suggestion
    public function testComponentPropsWithTrailingComma(): void
    {
        $output = self::getContainer()->get(Environment::class)->render('anonymous_component_props_trailing_comma.html.twig');
```

There are more places with this typo.

Comment (smnandre) [reply src/TwigComponent/tests/Integration/ComponentExtensionTest.php]: Fixed, thank you 🥲

**Review by WebMamba — APPROVED** [review]
Thanks @smnandre

**Comment by weaverryan** [comment]
Thank you Simon! That was annoying me too :)

**Pull request merged**